src/main/webapp/scripts/place-suggestions.js

```diff
@@ -0,0 +1,28 @@
+var map;
+var service;
+
+const MTV = new google.maps.LatLng(37.3861,122.0839);
+
+function findNearbyPlaces() {
+  map = new google.maps.Map(document.getElementById('map'), {
+      center: MTV,
+      zoom: 15
+    });
+
+  let request = {
+    location: MTV,
+    radius: '5000',
+    type: ['restaurant']
+  };
+
+  service = new google.maps.places.PlacesService(map);
+  service.nearbySearch(request, callback);
+}
+
+function callback(results, status) {
+  if (status == google.maps.places.PlacesServiceStatus.OK) {
```

Comment:
I think we should try our best to use `===` over `==` in all our JS code. It helps to reduce technical debt when we have a bunch of different types and yells at us if the two sides have some malformed type.